cypress/integration/languages.spec.ts

```diff
@@ -0,0 +1,64 @@
+describe("AutoComplete Component", () => {
+  beforeEach(() => {
+    cy.visit("/languages?");
```

Comment:
Why the empty query string?
```suggestion
    cy.visit("/languages");
```